t/ui/18-tests-details.t

```diff
@@ -563,6 +563,41 @@ subtest ansiToHtml => sub {
 
 my $t = Test::Mojo->new('OpenQA::WebAPI');
 
+subtest 'GRU dependency shown in infopanel' => sub {
+    $test_case->login($t, 'arthur');
```

Comment:
TODO: find why the login is not working here

Reply:
you are already logged in.
in NON_HEADLESS mode I can see the link.